system-addon/lib/TopSitesFeed.jsm

```diff
@@ -234,13 +231,10 @@ this.TopSitesFeed = class TopSitesFeed {
       case at.INIT:
         this.refresh();
         break;
-      case at.NEW_TAB_LOAD:
-        if (
-          // When a new tab is opened, if the last time we refreshed the data
-          // is greater than 15 minutes, refresh the data.
-          (Date.now() - this.lastUpdated >= UPDATE_TIME)
-        ) {
-          this.refresh(action.meta.fromTarget);
+      case at.SYSTEM_TICK:
+        if (Date.now() - this.lastUpdated >= UPDATE_TIME) {
+          // No update target, no broadcast.
```

Comment:
Like Highlights, remove the update time stuff. Also, comment is outdated and probably won't be necessary if we have `refresh({broadcast: false})`